### ISSUE TITLE
http 403 error `Website Unavailable`

### ISSUE THREAD

**Issue by xBZZZZ** [gd-programming/gddocs#124]
when I visit http://docs.gdprogra.me/#/ I see this:![Website Unavailable](https://user-images.githubusercontent.com/91378413/235297864-60be2219-8c21-4023-b421-21099775eb45.png)

**Comment by xBZZZZ** [comment]
now it's just loading forever

**Comment by HGStyle** [comment]
yes, thats because the site is now hosted on github, new link is:
https://docs.gd-programming.org/
you can find it on google by searching "gd docs", itll be in the first results

**Comment by xBZZZZ** [comment]
https://docs.gd-programming.org/ has a lot less info (nothing about level string, save files)

**Comment by Cvolton** [comment]
there's also this https://wyliemaster.github.io/gddocs/#/ but that's based on a fork and not this exact repo i believe (although that is actually a benefit because i believe it includes more information atp)

**Comment by HGStyle** [comment]
> there's also this https://wyliemaster.github.io/gddocs/#/ but that's based on a fork and not this exact repo i believe (although that is actually a benefit because i believe it includes more information atp)

thats why i usually go to the github repo (this one) and see the markdown docs in github... because actually the docs are here BUT the webserver hasn't been updated since a long time, and/or they have literally no link to the article in the articles menu

**Comment by xBZZZZ** [comment]
<blockquote cite="https://github.com/gd-programming/gd.docs/issues/124#issuecomment-1657063841">https://docs.gd-programming.org/ has a lot less info (nothing about level string, save files)</blockquote>nevermind, https://docs.gd-programming.org/ has all information in <a href="https://github.com/gd-programming/gd.docs/tree/docs"><code>docs</code> branch</a> except in <a href="https://github.com/gd-programming/gd.docs/tree/docs/docs/resources"><code>/docs/resources</code> directory</a> but not linked in left side

example: https://docs.gd-programming.org/client/gamesave/

links in left side seem to be generated from <a href="https://github.com/gd-programming/gd.docs/blob/docs/mkdocs.yml"><code>/mkdocs.yml</code> file</a>
